mod-mosaic/schemas/mosaic_order.json

```diff
@@ -0,0 +1,156 @@
+      "type": "string"
+    },
+    "referenceNumbers": {
+      "description": "Collection of reference number items",
+      "$ref": "../../common/schemas/reference_numbers.json"
+    },
+    "listUnitPrice": {
+      "description": "The per-item list price for physical or resources of 'Other' order format",
+      "type": "number"
+    },
+    "listUnitPriceElectronic": {
+      "description": "The e-resource per-item list price",
+      "type": "number"
+    },
+    "currency": {
+      "description": "An ISO currency code",
+      "type": "string"
+    },
+    "quantityPhysical": {
+      "description": "Quantity of physical items or resources of 'Other' order format in this purchase order line",
+      "type": "integer"
+    },
+    "quantityElectronic": {
+      "description": "Quantity of electronic items in this purchase order line",
+      "type": "integer"
+    },
+    "folioOrderTemplateId": {
+      "description": "UUID of the order template",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "locationId": {
+      "description": "location of the material",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "fundId": {
+      "description": "fund of poLine",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "expenseClassId": {
+      "description": "expense class of the material",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "materialTypeId": {
+      "description": "material type of the material",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "acquisitionUnitId": {
+      "description": "acquisition unit",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "billTo": {
+      "description": "UUID for the billing address",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "shipTo": {
+      "description": "UUID for the shipping address",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "vendor": {
+      "description": "UUID for the vendor organization",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "materialSupplier": {
+      "description": "UUID for the material supplier",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "accessProvider": {
+      "description": "UUID for the access provider",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "format": {
+      "description": "Order format (Physical, electronic, P/E mix)",
+      "type": "string",
+      "enum": [
+        "Physical",
+        "Electronic",
+        "P/E mix"
+      ]
+    },
+    "customFields": {
+      "description": "Custom fields for the order",
+      "type": "object",
+      "additionalProperties": {
+        "type": [
+          "string",
+          "null"
+        ]
+      }
+    },
+    "metadata": {
+      "type": "object",
+      "$ref": "../../../raml-util/schemas/metadata.schema",
+      "readonly": true
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "id"
```

Comment:
id also can be autogenerate in our side in case if it is not provided by Mosaic

Reply:
thanks added sample data